clang/lib/StaticAnalyzer/Checkers/VLASizeChecker.cpp

```diff
@@ -205,6 +199,12 @@ ProgramStateRef VLASizeChecker::checkVLAIndexSize(CheckerContext &C,
     State = StatePos;
   }
 
+  // Check if the size is tainted.
+  if ((StateNeg || StateZero) && isTainted(State, SizeV)) {
```

Comment:
At this point `StateNeg` may be default-initialized; but I assume that it doesn't cause any problems.